### PR TITLE
Added new option in VRT to Upload Baseline image

### PR DESCRIPTION
**Pull request by prashelke**
Thank you for your contribution.
Before submitting this PR, please make sure:

- [x] PR description and commit message should describe the changes done in this PR
- [x] Verify the PR is pointing to correct branch i.e. Release or Beta branch if the code fix is for specific release , else point it to master
- [x] Latest Code from master or specific release branch is merged to your branch
- [x] No unwanted\commented\junk code is included
- [x] No new warning upon build solution
- [x] Code Summary\Comments are added to my code which explains what my code is doing
- [ ] Existing unit test cases are passed
- [ ] New Unit tests are added for your development
- [ ] Sanity Tests are successfully executed for New and Existing Functionality
- [x] Verify that changes are compatible with all relevant browsers and platforms.
- [x] After creating pull request there should not be any conflicts
- [ ] Resolve all Codacy comments
- [ ] Builds and checks are passed before PR is sent for review
- [ ] Resolve code review comments
- [ ] Update the Help Library document to match any feature changes

## Summary by CodeRabbit

- **New Features**
	- Enhanced the Visual Testing (VRT) page with new UI elements such as CheckBox, RadioButtons, and ValueExpression for improved testing capabilities.
	- Introduced functionality for creating baselines and selecting images for Visual Regression Testing.
	- Added a new configuration note for manual VRT setup.
	- Improved Applitools integration with a dedicated configuration note and UI enhancements.

### CODE REVIEW

File: Ginger/Ginger/Actions/ActionEditPages/VisualTesting/VRTComparePage.xaml.cs [hunk 1]
```diff
@@ -144,6 +168,35 @@
...
+                        xPreviewImage.Visibility = Visibility.Visible;
+                    }
+                    else
+                    {
+                        VRTCurrentBaselineImagePathTxtBoxPnl.Visibility = Visibility.Collapsed;
+                        VRTBaseImageFramePnl.Visibility = Visibility.Collapsed;
+                        xBaselineImagePath.Visibility = Visibility.Collapsed;
+                        xPreviewImage.Visibility = Visibility.Collapsed;
+                    }
+                    if (xCreateBaselineCheckbox.IsChecked == true)
```
[the remark this answers is not in the document]
Comment (prashelke) [reply Ginger/Ginger/Actions/ActionEditPages/VisualTesting/VRTComparePage.xaml.cs:185]: updated

File: Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs [hunk 2]
```diff
@@ -35,6 +35,8 @@
         public static string VRTParamDiffTollerancePercent = "VRTParamDiffTollerancePercent";
         public static string VRTParamBuildName = "VRTParamBuildName";
         public static string ImageName = "ImageName";
+        public static string BaselineImage = "BaselineImage";
+        public static string VRTSavedBaseImageFilenameString = "VRTSavedBaseImageFilenameString";
```
[the remark this answers is not in the document]
Comment (prashelke) [reply Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs:39]: updated

File: Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs [hunk 3]
```diff
@@ -190,7 +200,23 @@
...
+                        {
+                            string baselinefilename = mAct.GetInputParamCalculatedValue(VRTSavedBaseImageFilenameString);
+                            image = GetBaseLineImage(baselinefilename);
+                        }
+                        mAct.CreateBaselineImage = false;//unchecked create Base line image after creation
+                    }
+                    else
+                    {
+                        image = mDriver.GetScreenShot(null, mAct.IsFullPageScreenshot);
+                    }
```
[the remark this answers is not in the document]
Comment (prashelke) [reply Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs:219]: done

File: Ginger/Ginger/Actions/ActionEditPages/VisualTesting/VRTComparePage.xaml.cs [hunk 4]
```diff
@@ -144,6 +169,35 @@
...
```
[the remark this answers is not in the document]
Comment (prashelke) [reply Ginger/Ginger/Actions/ActionEditPages/VisualTesting/VRTComparePage.xaml.cs:186]: done

File: Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (prashelke) [reply Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs:39]: done

**Review by Maheshkale447 — APPROVED** [review]

**Pull request merged**